packages/app/src/app/components/WorkspaceSetup/steps/Create.tsx

```diff
@@ -132,8 +132,7 @@ export const Create: React.FC<StepProps> = ({
           <Stack gap={2} direction="vertical">
             <Element css={{ position: 'relative', paddingBottom: '24px' }}>
               <InputText
-                label="Set a name for your workspace"
-                placeholder="My workspace"
+                label="Workspace name"
```

Comment:
isn't this a bit repetitive? we can also visually hide the label and it will by there for semantics and screen readers.